qiskit/visualization/matplotlib.py

```diff
@@ -366,11 +366,9 @@ def _get_gate_ctrl_text(self, op):
 
         if gate_text in self._style['disptex']:
             gate_text = "{}".format(self._style['disptex'][gate_text])
-        else:
-            gate_text = "{}".format(gate_text[0].upper() + gate_text[1:])
+        elif gate_text == op.name or gate_text == base_name:
```

Comment:
```suggestion
        elif gate_text in (op.name, base_name):
```
One last thing to fix the lint issue. (I sometimes revert to my C language past :))

Reply:
Done!